src/pages/Facility/settings/layout.tsx

```
@@ -1,5 +1,7 @@
-import { useRoutes } from "raviger";
 
+// create a layout for the facility settings page
+import { Link, navigate, useRoutes } from "raviger";
+import { useEffect } from "react";
```

ESLint

[error] Insert `⏎`

(prettier/prettier)